doc/association-rules.rst

```diff
@@ -8,13 +8,13 @@ Induction of association rules.
 Signals
 -------
 
-**Inputs**:
+#### Inputs
 
 -  **Data**
 
 Data set
```

Comment:
If you align these description lines with four spaces under the list item heading, does it become part of the item?